api/config/config_prod.json

```diff
@@ -108,11 +108,11 @@
     "useNewShibbolethService": true,
     "enableResearchPurposePrompt": false,
     "enableCohortBuilderV2": true,
-    "enableConceptSetSearchV2": false,
+    "enableConceptSetSearchV2": true,
     "enableReportingUploadCron": true,
-    "enableCOPESurvey": false,
+    "enableCOPESurvey": true,
```

Comment:
Do we still need this since we now use the cdr context instead? Looks like we no longer use this anywhere in the UI, not sure about API.

Reply:
You may be correct, but i say we leave it for now. We have several large cleanup task after all the new features are turned on in prod, so we can just clean it up then.

Reply:
SGTM